CHANGELOG.md

```diff
@@ -1,50 +1,65 @@
 # Backtrace Unity Release Notes
 
+## Version 3.11.0
+
+Improvements
+
+- Xbox One support (#230)
+
 ## Version 3.10.0
 
 Bugfixes
+
 - Fixed Unity console errors caused by corrupt meta (#224).
 - Fixed texture re-rendering during attachments generation (#228).
 
 Improvements
+
 - Mac native support (#225).
 - changed the frequency of the internet availability check to 30 seconds (#226).
 
 ## Version 3.9.1
 
 Bugfixes
+
 - Fixed potential overflow that can happen during breadcrumb deletion flow (#220).
 - Added proguard support for Android uncaught exception captured by the Unity uncaught exception handler (#221).
 
 Improvements
+
 - Native crash reporting binary upgrade - PlCrashReporter upgrade from 1.7.2 to 1.11.2 (#222).
 - iOS crash handler: store client attributes in the PlCrashReporter crashData object (#222).
-- iOS native attributes: Renaming process.vm.* attributes to vm.* (#222).
+- iOS native attributes: Renaming process.vm._ attributes to vm._ (#222).
```

Comment:
is this intentional? also, in the original version, the asterisks should be escaped (\\\*)